applications/app/views/fragments/indexBody.scala.html

```diff
@@ -3,10 +3,17 @@
 @import model.FrontProperties.empty
 @import services.IndexPage
 @import views.html.fragments.containers.facia_cards.container
+@import views.html.fragments.containers.facia_cards.containerScaffold
 @import views.html.fragments.pagination
 @import views.support.RenderClasses
 @import services.FaciaContentConvert
 
+@if(index.page.id == "help/accessibility-help") {
```

Comment:
yeah seems like this should be a `willLoad` property or something.

more importantly, we should really have a way of rendering react-based UIs on the server...